### ISSUE TITLE
BLE 4.2

### ISSUE THREAD

**Issue by tcpipchip** [Edragon/RDA#4]
Sir, can you explain how to compile one of BLE examples ?

**Comment by kissste** [comment]
I believe, these is no BLE (not even classic Bluetooth) on RDA5981 chip.

**Comment by tcpipchip** [comment]
Yes, RDA5981 has bluetooth! The sdk has has the ble includes! But i am having problem to compile!

**Comment by kissste** [comment]
I hope, I'm wrong. That would great.

RDA5856 does have a Bluetooth, the SDK might be "generic" to all of their chips.

mbedos does have a BLE support are is there a "target" code for RDA5981? 

What's the error message that are you getting?

Updating... looks like RDA5981C has got BLE.

**Comment by tcpipchip** [comment]
Humm, make sense!

**Comment by tcpipchip** [comment]
I think that you are correct!
